lib/LaTeXML/Package/listings.sty.ltxml

```diff
@@ -864,7 +868,7 @@ DefKeyVal('LST', 'numberstyle',      '');
 DefKeyVal('LST', 'numbersep',        'Dimension');
 DefKeyVal('LST', 'numberblanklines', '', 'true');
 DefKeyVal('LST', 'firstnumber',      '');
-DefKeyVal('LST', 'name',             '');
+DefKeyVal('LST', 'name',             'Semiverbatim');
```

Comment:
This should be reverted I think.

Reply:
should be harmless, though, and is maybe a good hint :>